src/base/utils/gzip.cpp

```diff
@@ -27,116 +27,121 @@
  * exception statement from your version.
  */
 
+#include "gzip.h"
+
 #include <QByteArray>
-#include <zlib.h>
 
-#include "gzip.h"
+#define ZLIB_CONST  // make z_stream.next_in const
```

Comment:
To avoid a possible compiler warning, I would protect this with #ifdef:
```C++
#ifndef ZLIB_CONST
#define ZLIB_CONST
#endif

Perhaps the define comes from a dependent package via pkg-config or by another way?